common.go

```diff
@@ -29,70 +29,69 @@ type UserStore interface {
 }
 
 // Config is a generic map of services/methods to a config value.
+// map[service]map[method]T
 type Config[T any] map[string]map[string]T
 
-// Get returns the config value for the given request.
-func (c Config[T]) Get(r *rcpRequest) (v T, ok bool) {
-	if c == nil || r.Package != "rpc" {
+// returns the config value for the given request.
+func (c Config[T]) get(r *rcpRequest) (v T, ok bool) {
```

Comment:
Can you keep this public?
I use it here: https://github.com/0xsequence/quotacontrol/pull/53/files#diff-7bf08ce0a790451f4677b47d7e17aafcc1825aa92312c83775de6c22e7b5a102R10-R23